examples/visualization/plot_epochs.py

```diff
@@ -0,0 +1,52 @@
+"""
+=============================================================
+Demonstration for plotting and rejecting epochs interactively
+=============================================================
+
+This example shows how to visualize and reject bad epochs interactively.
+Bad epochs can be selected by a left click on the top of the epoch.
+The selected bad epochs are dropped on closing of the figure.
+Scaling can be changed runtime by pressing page up and page down keys.
+"""
+
+# Authors: Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
```

Comment:
if there's already an example, can you just update the old example instead of adding a new one?